extensions/community/PokiGamesSDK.json

```diff
@@ -0,0 +1,321 @@
+  "name": "PokiGamesSDK",
+  "previewIconUrl": "",
+  "shortDescription": "Extension that implements Poki SDK so that your game can be hosted on Poki.",
+  "version": "0.0.1",
+  "description": [
+    "Please refer to [Poki's guide](https://sdk.poki.com/sdk-documentation.html) to know when to call the different actions.",
+    "",
+    "To initialize the SDK, use the asynchronous action \"Load SDK\" at the beginning of your very first scene."
+  ],
+  "tags": [
+    "poki",
+    "ads",
+    "sdk"
+  ],
+  "authorIds": [
+    "tgIIofwSWVNtHsFW7AR2imu8gCj1"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "fullName": "",
+      "functionType": "Action",
+      "name": "onFirstSceneLoaded",
+      "sentence": "",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "gdjs._pokiGamesSdkExtension = {};",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "async": true,
+      "description": "Load Poki SDK.",
+      "fullName": "Load SDK",
+      "functionType": "Action",
+      "group": "Poki Games",
+      "name": "LoadSDK",
+      "sentence": "Load Poki SDK",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "function addScript(src) {",
+            "    return new Promise((resolve, reject) => {",
+            "        const scriptElement = document.createElement('script');",
+            "",
+            "        scriptElement.setAttribute('src', src);",
+            "        scriptElement.addEventListener('load', resolve);",
+            "        scriptElement.addEventListener('error', reject);",
+            "",
+            "        document.body.appendChild(scriptElement);",
+            "    });",
+            "}",
+            "addScript('https://game-cdn.poki.com/scripts/v2/poki-sdk.js').then(() => {",
+            "    PokiSDK.init().then(() => {",
+            "        console.log(\"Poki SDK successfully initialized\");",
```

Comment:
If you ignore my advice and keep this, at least do not use console.log, use gdjs.Logger.

Reply:
Oh yeah, missed that!